tests/foreman/remote_DB/test_remote_DB.py

```diff
@@ -0,0 +1,228 @@
+"""Test class for Remote DB feature
+
+:Requirement: Offload DBs from Satellite Server
+
+:CaseAutomation: ManualOnly
```

Comment:
I think these tests are automated right ? The automation code in not in robottelo ?

Reply:
These are not in robotello and never will be - currently it's jenkins job that trigger beaker job. So It's kind of automated but not the same as the rest of our automation.